test/integration/default/inspec.yml

```diff
@@ -0,0 +1,12 @@
+name: locale
+title: Locale Formula
+maintainer: Nicolas Rodriguez
```

Comment:
don't you think 'Saltstack Formulas' is better suited?

Reply:
for the maintainer?

Reply:
yes, as the community is the maintainer of the formulas, it seems logical for me